doc/getting_started/index.md

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
Please as many as possible review this update to the getting started *definition*. Its important we agree on the purpose and scope of this guide.

- Should it be renamed to something with *tour*? "Take the tour". Because that is the purpose.

@philippjfr